packages/Drawer/__tests__/__snapshots__/Drawe.test.tsx.snap

```diff
@@ -1,7 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
type on file name, should be Drawer, not Drawe

Reply:
Renamed